etc/sysctl/90-crimson-osd.conf.in

```diff
@@ -0,0 +1,3 @@
+fs.aio-max-nr = 2097152
```

Comment:
I'm not sure if we should keep 1048576 or increase it since there were a few deployments cases where the lower value was not enough.

Reply:
2097152 looks fine to me. I always increase it when OSD startup is failed -- now it is 4194304 in my local env.